Contents.swift

```diff
@@ -1,11 +1,30 @@
 import Cocoa
 
+// create an array of various shapes
 let shapes: Array<Shape> = [
     Rectangle(withHeight: 1, withWidth: 2),
-    Rectangle(withHeight: 2, withWidth: 3),
-    Rectangle(withHeight: 3, withWidth: 4)
+    Triangle(withSideA: 5, withSideB: 5, withBase: 6, withHeight: 4),
+    Square(withSide: 4),
 ]
 
+// loop through each shape, print its area, scale, then print its new area
 for shape in shapes {
+    try shape.printAreaToConsole()
     shape.scale(withScaleFactor: 10)
+    try shape.printAreaToConsole()
+}
+
+// create an array of rectangles
+let rectangles: Array<Rectangle> = [
```

Comment:
Are you following a coding style that requires variables to be created at the top of the scope, or is this acceptable?